README.md

```diff
@@ -58,72 +57,107 @@ Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP
 4. Change group ownership of repo (needs to be redone when a new file occurs in the repo):
     - Linux: `chgrp -R fim <path/to/repository>`
 
+### Folder Structure
+You are welcome to set up your folder structure in any pattern you like. For example purposes, we will use a folder structure shown below.
+Starting with a base folder, e.g `/home/my_user/projects/` add the following folders:
+- `fim`
+   - `code`
+   - `data`
+      - `inputs`
+      - `outputs`
+      - `outputs_temp`
+
+### Input Data
+Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP S3 Bucket" section above). The FIM inputs directory can be found at `s3://noaa-nws-owp-fim/hand_fim/inputs`. It is appx 400GB and it needs to be in your `data` folder.
+
+```
+aws s3 cp --recursive s3://noaa-nws-owp-fim/hand_fim/inputs /home/my_user/projects/fim/data/inputs --request-payer requester --dry-run
+```
+**Note**: When you include the `--dry-run` argument in the command, a large list will be returned showing you exactly which files are to be downloaded and where they will be saved. We recommend including this argument the first time you run the command, then quickly aborting it (CTRL-C) so you don't get the full list. However, you can see that your chosen target path on your machine is correct.  When you are happy with the pathing, run the `aws s3` command again and leave off the `--dry-run` argument.
+
+The S3 inputs directory has all of the folders and files you need to run FIM. It includes some publicly available and some non-publicly availible data.
+
+### Getting FIM source code
+(based on sample pathing described above)
+
+``` path to your "code" folder. e.g.
+cd /home/my_user/projects/code
```

Comment:
We need to insert the `fim` directory in this command to match the example folder structure listed above: `cd /home/my_user/projects/fim/code`